src/webgl/3d_primitives.js

```diff
@@ -1010,30 +1010,51 @@ p5.RendererGL.prototype.rect = function(args) {
   return this;
 };
 
-p5.RendererGL.prototype.quad = function(x1, y1, x2, y2, x3, y3, x4, y4) {
+p5.RendererGL.prototype.quad = function(
+  x1,
+  y1,
+  z1,
+  x2,
+  y2,
+  z2,
+  x3,
+  y3,
+  z3,
+  x4,
+  y4,
+  z4
+) {
```

Comment:
linter/prettier made me format args like this ^

Reply:
You can use `// prettier-ignore` above a line when you feel confident that it is easier to read without the prettier modifications.